plugins/action_change_status/src/changeStatus.ts

```diff
@@ -115,24 +115,26 @@ const changeReviewStatus = async (
 ): Promise<ActionPluginOutput> => {
   const returnObject: ActionPluginOutput = {
     status: ActionQueueStatus.Fail,
-    error_log: 'uknown error',
+    error_log: '',
   }
   console.log(`Changing the Status of Review ${reviewId}...`)
 
   try {
     const currentStatus = await DBConnect.getReviewCurrentStatusHistory(reviewId)
 
+    console.log('currentStatus', currentStatus)
+
     if (currentStatus?.status === newStatus) {
       // Do nothing
       console.log(
         `WARNING: Review ${reviewId} already has status: ${newStatus}. No changes were made.`
       )
       returnObject.status = ActionQueueStatus.Success
-      returnObject.error_log = 'Status not changed'
+      returnObject.error_log = 'Review Status not changed'
       returnObject.output = {
         status: newStatus,
         statusId: currentStatus.status_history_id,
-        reviewStatusHistoryTimestamp: currentStatus.status_history_time_created,
+        reviewStatusHistoryTimestamp: currentStatus.time_created,
```

Comment:
This was obviously copied from the application version without checking the database script -- the field name is different.